### DISCUSSION TITLE
Lighthouse does not show a result

### DISCUSSION THREAD

**Discussion by reeceatkinson** [GoogleChrome/lighthouse#14310] [Q&A] [answered]
I tried to run a lighthouse test to check performance of the website. But it just shows a question mark. It works on all other pages of the site but not this one page.
![image](https://user-images.githubusercontent.com/78908925/185735371-428ab610-f218-429b-b3f0-134a0610ab6c.png)

**Comment by adamraine** [comment, marked as answer]
You need an LCP for Lighthouse to calculate a performance score. Judging by the page thumbnail, the page probably doesn't paint any element that qualifies as an LCP.

For more info: https://web.dev/lcp/#what-elements-are-considered
